src/NuGet.Core/NuGet.Build.Tasks.Pack.Library/NuGet.Build.Tasks.Pack.Library.csproj

```diff
@@ -94,6 +95,13 @@
       </DllsInOutputDir>
     </ItemGroup>
   </Target>
+
+  <!--These targets help get all the DLLs that are packed in the nuspec to be sent for signing.-->
+  <Target Name="GetIlmergeSymbolOutput" Returns="@(SymbolsToIndex)">
+    <ItemGroup>
+      <SymbolsToIndex Include="$(OutputPath)ilmerge\*.pdb" KeepDuplicates="false" />
```

Comment:
the merged DLL is already coming in through the GetIlMergeBuildOutput. (https://github.com/NuGet/NuGet.Client/pull/1344/files#diff-e3360b39dbc9487df71573c870a8efccR108)